tutorials/turbo_1/turbo_1.ipynb

```diff
@@ -802,267 +719,1350 @@
-    "    train_Y = (Y_ei - Y_ei.mean()) / Y_ei.std()\n",
-    "    likelihood = GaussianLikelihood(noise_constraint=Interval(1e-8, 1e-3))\n",
-    "    model = SingleTaskGP(X_ei, train_Y, likelihood=likelihood)\n",
-    "    mll = ExactMarginalLogLikelihood(model.likelihood, model)\n",
-    "    fit_gpytorch_mll(mll)\n",
-    "\n",
-    "    # Create a batch\n",
-    "    ei = qExpectedImprovement(model, train_Y.max())\n",
-    "    candidate, acq_value = optimize_acqf(\n",
-    "        ei,\n",
-    "        bounds=torch.stack(\n",
-    "            [\n",
-    "                torch.zeros(dim, dtype=dtype, device=device),\n",
-    "                torch.ones(dim, dtype=dtype, device=device),\n",
-    "            ]\n",
-    "        ),\n",
-    "        q=batch_size,\n",
-    "        num_restarts=NUM_RESTARTS,\n",
-    "        raw_samples=RAW_SAMPLES,\n",
-    "    )\n",
-    "    Y_next = torch.tensor(\n",
-    "        [eval_objective(x) for x in candidate], dtype=dtype, device=device\n",
-    "    ).unsqueeze(-1)\n",
-    "\n",
-    "    # Append data\n",
-    "    X_ei = torch.cat((X_ei, candidate), axis=0)\n",
-    "    Y_ei = torch.cat((Y_ei, Y_next), axis=0)\n",
-    "\n",
-    "    # Print current status\n",
-    "    print(f\"{len(X_ei)}) Best value: {Y_ei.max().item():.2e}\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "originalKey": "38f8ac21-d9ae-41f7-ba42-0ff6abde0a2c",
-    "showInput": false
-   },
-   "source": [
-    "## Sobol"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 10,
-   "metadata": {
-    "collapsed": false,
-    "executionStartTime": 1674921754972,
-    "executionStopTime": 1674921755010,
-    "jupyter": {
-     "outputs_hidden": false
     },
-    "originalKey": "a6333e87-1fcf-4174-9cb8-111598dd7780",
-    "requestMsgId": "629a258e-fa1d-44f8-848c-3335a98e3421"
-   },
-   "outputs": [],
-   "source": [
-    "X_Sobol = (\n",
-    "    SobolEngine(dim, scramble=True, seed=0)\n",
-    "    .draw(len(X_turbo))\n",
-    "    .to(dtype=dtype, device=device)\n",
-    ")\n",
-    "Y_Sobol = torch.tensor(\n",
-    "    [eval_objective(x) for x in X_Sobol], dtype=dtype, device=device\n",
-    ").unsqueeze(-1)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "originalKey": "e20c8975-af02-4308-a1ef-3f12afb85ffd",
-    "showInput": false
-   },
-   "source": [
-    "## Compare the methods"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 11,
-   "metadata": {
-    "collapsed": false,
-    "executionStartTime": 1674921755158,
-    "executionStopTime": 1674921757156,
-    "jupyter": {
-     "outputs_hidden": false
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/radical-rhys/Radical/botorch/botorch/acquisition/monte_carlo.py:396: NumericsWarning: qExpectedImprovement has known numerical issues that lead to suboptimal optimization performance. It is strongly recommended to simply replace\n",
+      "\n",
```

Comment:
This is spamming a lot of the notebook outputs. Let's silence this specific warning with a context manager (and leave a comment why), so this is easier readable.